heartbeat/monitors/logger/logger.go

```diff
@@ -39,13 +39,16 @@ type DurationLoggable struct {
 	Mills int64 `json:"ms"`
 }
 
+type NetworkInfo map[string]interface{}
+
 type MonitorRunInfo struct {
-	MonitorID string `json:"id"`
-	Type      string `json:"type"`
-	Duration  int64  `json:"-"`
-	Steps     *int   `json:"steps,omitempty"`
-	Status    string `json:"status"`
-	Attempt   int    `json:"attempt"`
+	MonitorID   string      `json:"id"`
+	Type        string      `json:"type"`
+	Duration    int64       `json:"-"`
+	Steps       *int        `json:"steps,omitempty"`
+	Status      string      `json:"status"`
+	Attempt     int         `json:"attempt"`
+	NetworkInfo NetworkInfo `json:"network_info,omitempty"`
```

Comment:
The new logged fields are ECS complaint fields, In order to keep it simple, It would be better to add them to the top level instead of adding under `monitor.*`. These would not require any new mapping when we use it for debugging purposes.

Reply:
@vigneshshanmugam TIL, can you share a link to ECS spec where these are included? Can't seem to find it

Reply:
My bad, They are indeed HB specific - https://www.elastic.co/guide/en/beats/heartbeat/current/exported-fields-tls.html#_handshake and not mentioned in the ECS docs. In this case, I dont have any strong opinion on having them as root or non-root. Thanks for catching Emilio.

Reply:
I see, thanks both for checking on this!